cache/purge.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ *
+ * phpBB Media Embed PlugIn extension for the phpBB Forum Software package.
+ *
+ * @copyright (c) 2020 phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ */
+
+namespace phpbb\mediaembed\cache;
+
+use \Symfony\Component\Finder\Finder;
+
+/**
+ * mediaembed cron task.
+ */
+class purge
+{
+	/** @var \phpbb\cache\driver\driver_interface */
+	protected $cache;
+
+	/**
+	 * Constructor
+	 *
+	 * @param \phpbb\cache\driver\driver_interface  $cache      Cache jbject
+	 * @access public
+	 */
+	public function __construct(\phpbb\cache\driver\driver_interface $cache)
+	{
+		$this->cache = $cache;
+		$this->finder = new Finder();
```

Comment:
`$this->finder` property is not defined. Also maybe we don't need to. Just instantiate it in the purge class.